src/algebraic_topology/simplex_category.lean

```diff
@@ -402,4 +402,149 @@ full_subcategory_inclusion _
 
 end truncated
 
+section concrete
+
+instance : concrete_category.{0} simplex_category.{u} :=
+{ forget :=
+  { obj := λ i, fin (i.len + 1),
+    map := λ i j f, f.to_preorder_hom },
+  forget_faithful := {} }
+
+end concrete
+
+section epi_mono
+
+/-- A morphism in `simplex_category` is a monomorphism precisely when it is an injective function
+-/
+theorem mono_iff_injective {n m : simplex_category} {f : n ⟶ m} :
+  mono f ↔ function.injective f.to_preorder_hom :=
+begin
+  split,
+  { introsI m x y h,
+    have H : const n x ≫ f = const n y ≫ f,
+    { dsimp, rw h },
+    change (n.const x).to_preorder_hom 0 = (n.const y).to_preorder_hom 0,
+    rw cancel_mono f at H,
+    rw H },
+    { intro hyp_inj,
+      exact concrete_category.mono_of_injective _ hyp_inj }
+end
+
+/-- A morphism in `simplex_category` is an epimorphism if and only if it is a surjective function
+-/
+lemma epi_iff_surjective {n m : simplex_category} {f: n ⟶ m} :
+  epi f ↔ function.surjective f.to_preorder_hom :=
+begin
+  split,
+  { intro hyp_f_epi,
+    intro x,
+    by_contradiction h_ab,
+    rw not_exists at h_ab,
+    -- The proof is by contradiction: assume f is not surjective,
+    -- then introduce two non-equal auxiliary functions equalizing f, and get a contradiction.
+    -- First we define the two auxiliary functions.
+    set chi_1 : m ⟶ [1] := hom.mk
+    ⟨λ u, if u ≤ x then 0 else 1,
+     by { intros a b a_leq_b,
+          by_cases a ≤ x,
+          { simp [h], obviously },
+          { simp [h],
+            have b_ge_x : ¬(b ≤ x),
+            { by_contra b_leq_x,
+              have a_leq_x : a ≤ x,
+              { transitivity b,
+                exact a_leq_b,
+                exact b_leq_x },
+                exact h a_leq_x },
+          simp [b_ge_x]}}⟩ with def_chi_1,
```

Comment:
This can both avoid the non-terminal `simp` and prove the intermediate step more quickly:
```suggestion
          { have b_ge_x : ¬(b ≤ x),
            { contrapose! h,
              exact a_leq_b.trans h },
            simp [h, b_ge_x] }}⟩ with def_chi_1,
```